web/html/src/components/pagination.tsx

```diff
@@ -52,32 +52,48 @@ const PaginationBlock = (props: PaginationBlockProps) => {
 type PaginationButtonProps = {
   disabled?: boolean;
   onClick: (...args: any[]) => any;
-  text: React.ReactNode;
+  text?: React.ReactNode;
+  icon?: React.ReactNode;
 };
+const PaginationButton = (props: PaginationButtonProps) => {
+  return (
 
-const PaginationButton = (props: PaginationButtonProps) => (
-  <button type="button" className="btn btn-default" disabled={props.disabled} onClick={props.onClick}>
-    {props.text}
-  </button>
-);
+    <button type="button" className="btn btn-tertiary" disabled={props.disabled} onClick={props.onClick}>
+      <i className={`pagination-icon fa ${props.icon}`} />{props.text}
+    </button>
+  )
+};
 
 type ItemsPerPageSelectorProps = {
   currentValue: number;
   onChange: (value: number) => any;
+  itemCount: number;
+  fromItem: number;
+  toItem: number;
 };
 
 const ItemsPerPageSelector = (props: ItemsPerPageSelectorProps) => (
-  <select
-    className="display-number"
-    defaultValue={props.currentValue}
-    onChange={(e) => props.onChange(parseInt(e.target.value, 10))}
-  >
-    {[5, 10, 15, 25, 50, 100, 250, 500].map((o) => (
-      <option value={o} key={o}>
-        {o}
-      </option>
-    ))}
-  </select>
+  <div>
+    <DropdownButton
+      text={t("{from} - {to} of {total} Items", { from: props.fromItem, to: props.toItem, total: props.itemCount })}
```

Comment:
Not sure, but at a glance, it looks like using lowercase "items" would fit better with the rest of the text on the page?